### PR TITLE
subprocess_run to append and not replace variables to the external env

### PR DESCRIPTION
**Pull request by mpagot**

### CODE REVIEW

File: scripts/qesap/qesap.py [hunk 1]
```diff
@@ -63,10 +63,14 @@ def subprocess_run(cmd, env=None):
         return (1, [])
 
     log.info("Run:       '%s'", ' '.join(cmd))
+
+    external_env = dict(os.environ)
     if env is not None:
         log.info("with env %s", env)
+        # external_env |= env supported from python 3.9
+        external_env = {**external_env, **env}
```
Comment (ricardobranco777) [inline scripts/qesap/qesap.py:71]:
```suggestion
        external_env = external_env.update(env) 
```

**Review by ricardobranco777 — APPROVED** [review]
Small suggestion. Otherwise LGTM

File: scripts/qesap/qesap.py (hunk 1 above)
Comment (mpagot) [reply scripts/qesap/qesap.py:71]: indeed ... https://peps.python.org/pep-0584/#dict-update
ok let me check

**Review by alvarocarvajald — APPROVED** [review]
LGTM

File: scripts/qesap/qesap.py (hunk 1 above)
Comment (ricardobranco777) [reply scripts/qesap/qesap.py:71]: Updated suggestion

**Review by lpalovsky — APPROVED** [review]
LGTM

**Comment by mpagot** [comment]
No more relevant, superseded by https://github.com/SUSE/qe-sap-deployment/pull/94

**Pull request closed without merging**